tests/evaluate/test_default_evaluator.py

```diff
@@ -2095,7 +2103,12 @@ def test_evaluate_text_summarization_with_targets_no_type_hints():
     assert "eval_results_table.json" in artifacts
     logged_data = pd.DataFrame(**results.artifacts["eval_results_table"].content)
     pd.testing.assert_frame_equal(logged_data, data.assign(outputs=["a", "b"]))
-    assert results.metrics == {"rouge1": 1.0, "rouge2": 0.0, "rougeL": 1.0, "rougeLsum": 1.0}
+    assert set(results.metrics.keys()) == set(["rouge1", "rouge2", "rougeL", "rougeLsum", "mean_perplexity", "percent_toxic"])
+    assert results.metrics["rouge1"] == 1.0
+    assert results.metrics["rouge2"] == 0.0
```

Comment:
This sounds like a false positive in ruff. Let me create an issue in https://github.com/astral-sh/ruff